### PR TITLE
OCD-3519 Revise G1-G2 entry and display

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: dev/openchpl_data-model.sql [hunk 1]
```diff
@@ -3281,6 +3281,188 @@ create table openchpl.change_request_developer_details (
         MATCH SIMPLE ON UPDATE NO ACTION ON DELETE RESTRICT
 );
 
+CREATE TABLE openchpl.svap (
```
Comment (andlar) [inline dev/openchpl_data-model.sql]: SVAP stuff shouldn't be in this PR, right?

File: dev/openchpl_soft-delete.sql [hunk 2]
```diff
@@ -15,6 +15,13 @@ BEGIN
     UPDATE openchpl.listing_to_listing_map as src SET deleted = NEW.deleted WHERE src.parent_listing_id = NEW.certified_product_id;
     UPDATE openchpl.listing_to_listing_map as src SET deleted = NEW.deleted WHERE src.child_listing_id = NEW.certified_product_id;
 	UPDATE openchpl.quarterly_report_excluded_listing_map as src SET deleted = NEW.deleted WHERE src.listing_id = NEW.certified_product_id;
+    UPDATE openchpl.certified_product_mips_measure as src SET deleted = NEW.deleted WHERE src.certified_product_id = NEW.certified_product_id;
```
Comment (andlar) [inline dev/openchpl_soft-delete.sql]: We don't have "mips" in the tables any more, correct?

File: ocd-3519.sql [hunk 3]
```diff
@@ -0,0 +1,558 @@
...
+WHERE NOT EXISTS (
+    SELECT * 
+    FROM openchpl.questionable_activity_trigger
+    WHERE name = 'G1/G2 Removed'
+    AND level = 'Listing');
+    
+------------------- INSERT MIPS_DOMAINS -------------------
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'EC', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EH');
```
Comment (andlar) [inline ocd-3519.sql]: The first one is `EC`; the second is `EH`. They should be identical, right?

File: ocd-3519.sql [hunk 4]
```diff
@@ -0,0 +1,558 @@
...
+SELECT 'EH/CAH Medicaid PI', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EH/CAH Medicaid PI');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'EC Group', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EC Group');
+
+INSERT INTO openchpl.measure_domain
+(domain, last_modified_user)
+SELECT 'EC Individual (TIN/NPI)', -1
+WHERE NOT EXISTS (SELECT * FROM openchpl.measure_domain WHERE domain = 'EC Individual (TIN/NPI)');
```
Comment (andlar) [inline ocd-3519.sql]: Any particular reason this is on a newline?

File: ocd-3519.sql [hunk 5]
```diff
@@ -0,0 +1,558 @@
...
+INSERT INTO openchpl.measure_type
+(name, last_modified_user)
+SELECT 'G1', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_type WHERE name = 'G1');
+
+INSERT INTO openchpl.measure_type
+(name, last_modified_user)
+SELECT 'G2', -1 WHERE NOT EXISTS (SELECT * FROM openchpl.measure_type WHERE name = 'G2');
+
+------------------- INSERT MIPS_MEASURES -------------------
+INSERT INTO openchpl.measure (measure_domain_id, required_test_abbr, measure_name, required_test, criteria_selection_required, removed, last_modified_user) SELECT (SELECT id FROM openchpl.measure_domain WHERE domain = 'EC ACI Transition'), 'RT1', 'Electronic Prescribing: Eligible Clinician', 'Required Test 1: Promoting Interoperability Transition Objective 2 Measure 1 ', false, true, -1;
```
Comment (andlar) [inline ocd-3519.sql]: The "required_test" here has a space at the end of it. Is that required?

Comment (andlar) [reply ocd-3519.sql]: Ctrl-F and then ` ',` finds four others in the "insert MIPS measures" section

Oh, that comment could be updated too, though not required

File: ocd-3519.sql [hunk 6]
```diff
@@ -0,0 +1,558 @@
...
+INSERT INTO openchpl.measure (measure_domain_id, required_test_abbr, measure_name, required_test, criteria_selection_required, removed, last_modified_user) SELECT (SELECT id FROM openchpl.measure_domain WHERE domain = 'EC Group'), 'RT2', 'Patient Electronic Access: Eligible Clinician Group', 'Required Test 2: Stage 2 Objective 8 Measure 1 and Stage 3 Objective 5 Measure 1, ACI Transition Objective 3 Measure 1 and ACI Objective 3 Measure 1', true, false, -1;
+--INSERT INTO openchpl.measure (measure_domain_id, required_test_abbr, measure_name, required_test, criteria_selection_required, removed, last_modified_user) SELECT (SELECT id FROM openchpl.measure_domain WHERE domain = 'EH/CAH Medicare and Medicaid PI'), 'RT2', 'Provide Patients Electronic Access to Their Health Information (formerly Patient Electronic Access): Eligible Clinician', 'Required Test 2: Medicare and Medicaid Promoting Interoperability Programs', true, false, -1;
```
Comment (andlar) [inline ocd-3519.sql]: Curious about these commented out ones

**Review by andlar — CHANGES_REQUESTED** [review]

File: ocd-3519.sql (hunk 6 above)
Comment (tmy1313) [reply ocd-3519.sql]: Those were the measures that were duplicates, but had different names.  I have removed them from the script.

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by andlar — APPROVED** [review]

**Review by kekey1 — APPROVED** [review]

**Pull request merged**